chunky/src/java/se/llbit/chunky/entity/BeaconBeam.java

```diff
@@ -65,12 +70,108 @@ public BeaconBeam(JsonObject json) {
 
     JsonObject materialsList = json.get("beamMaterials").object();
     for (JsonMember obj : materialsList.members) {
-      BeaconBeamMaterial mat = new BeaconBeamMaterial(0xFFFFFF);
+      BeaconBeamMaterial mat = new BeaconBeamMaterial(0xF9FFFE);
```

Comment:
Why is this not white anymore?

Reply:
I changed it to use the actual values that Minecraft uses to tint the beam, which isn't pure white.
https://minecraft.gamepedia.com/Dye#Color_values

Reply:
`private static final int DEFAULT_BEAM_COLOR`?

Reply:
Yeah, in the last commit, it uses a constant in BeaconBeamMaterial